### ISSUE TITLE
push-button panes ignore width and height

### ISSUE THREAD

**Issue by jakykong** [robert-strandh/McCLIM#144]
It appears that creating a push-button pane with width and height, the width and height are ignored. I talked briefly with jackdaniel on #clim and he suggested :y-spacing/:x-spacing as a workaround. 

Here is a minimal example demonstrating the problem. The expected outcome is that the button labeled "Huge Button" should be taller than my screen. The actual outcome is it is about 32px tall. (Width is similar - it is actually a couple hundred px wide, instead of 60.)

```
(define-application-frame bug-app ()
  ()
  (:panes 
    (app :application)
    (btn (make-pane 'push-button 
                       :height 1000 :width 60 
                       :label "Huge Button")))
  (:layouts (default (vertically () app btn))))

(defun bug-main ()
  (run-frame-top-level (make-application-frame 'bug-app)))
```

**Comment by gas2serra** [comment]
The problem is that push-button set 'max-height' to the exact size.

@dkochmanski do you think we can set it to +fill+?

```
(defmethod compose-space ((gadget push-button-pane) &key width height)
  (declare (ignore width height))
  (space-requirement+* (space-requirement+* (compose-label-space gadget)
                                            :min-width (* 2 (pane-x-spacing gadget))
                                            :width (* 2 (pane-x-spacing gadget))
                                            :max-width (* 2 (pane-x-spacing gadget))
                                            :min-height (* 2 (pane-y-spacing gadget))
                                            :height (* 2 (pane-y-spacing gadget))
                                            :max-height (* 2 (pane-y-spacing gadget)))				    
					    )
                       :min-width (* 2 *3d-border-thickness*)
                       :width (* 2 *3d-border-thickness*)
                       :max-width (* 2 *3d-border-thickness*)
                       :min-height (* 2 *3d-border-thickness*)
                       :height (* 2 *3d-border-thickness*)
                       :max-height (* 2 *3d-border-thickness*)))
```

**Comment by dkochmanski** [comment]
fixed by #222.
